web/webpack.assets.json

```diff
@@ -19,6 +19,15 @@
         "./src/billing/sponsorship",
         "./styles/portico/billing.css"
     ],
+    "billing_auth": [
+        "./src/bundles/portico",
+        "./src/portico/tippyjs",
+        "./src/billing/helpers",
+        "jquery-validation",
+        "./src/billing/remote_billing_auth",
+        "./src/billing/deactivate_server",
```

Comment:
Does the `upgrade` bundle not need these two files?

Reply:
Yeah, updated.